frappe/printing/doctype/print_format/print_format.js

```diff
@@ -39,7 +39,7 @@ frappe.ui.form.on("Print Format", {
 			} else if (frm.doc.custom_format && !frm.doc.raw_printing) {
 				frm.set_df_property("html", "reqd", 1);
 			}
-			if (frappe.model.can_read(frm.doc.doc_type)) {
+			if (frappe.model.can_read("DocType")) {
```

Comment:
```suggestion
			if (frappe.model.can_write("Customize Form")) {
```


This is more appropriate? r/w doctype is only required in developer mode, far more common use case is changing default in production.